Fortran/exercise2/league_sim_mod.f90

```diff
@@ -0,0 +1,85 @@
+! module for league simulation routines
+module league_simulator
+
+implicit none
+
+integer :: max_goals = 5
```

Comment:
What is the purpose of this integer? Can you add a comment explaining it?

Reply:
If there is no specific reason for setting this to 5, suggest we allow the user to vary the max goals setting (with some sensible default).